components/UploadedFilePreview.js

```diff
@@ -159,17 +159,13 @@ const UploadedFilePreview = ({
       >
         {content}
       </CardContainer>
-      {showFileName && (
+      {caption && (
         <Box mt="6px" maxWidth={100}>
           {isLoading ? (
             <LoadingPlaceholder height={12} />
-          ) : fileName ? (
-            <P fontSize="12px" color="black.600" fontWeight="500">
-              {fileName}
-            </P>
           ) : (
-            <P fontStyle="italic" fontSize="12px" color="black.600">
-              <FormattedMessage id="File.NoFilename" defaultMessage="No filename" />
```

Comment:
Also to add to what Ben said above, whenever we make changes to the internationalized strings we also need to run `npm run langs:update` to update the language files. The reason the "check-langs" test is failing is due to this. 🙂

Reply:
Thank you @SudharakaP, hahah I was tinkering and figured this out the hard way. I'm also playing around with different switches that capture the text for translation, vs. passing params that apparently don't. Another commit is imminent

Reply:
@SudharakaP, the fix is here but I'm still not quite sure that it will translate properly. Can you advise?

Reply:
`check-langs` is now passing, so the lang files are updated correctly. So it seems all good here; https://github.com/opencollective/opencollective-frontend/runs/5749375411?check_suite_focus=true.

Reply:
Ah ok, I was confused because in those files the new labels don't seem to translate. Is this expected behavior or am I missing something?

Reply:
That's fine. It just adds the labels, the actual transactions are done through crowdin; https://crowdin.com/project/opencollective and needs to be manually verified.

Reply:
Aha, thanks for this! It's much clearer now.

I will go ahead and mark this discussion resolved.